elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -509,6 +508,12 @@ protected boolean updateToManyRelation(String fieldName,
                 });
 
 
+        //Hook for setRelation
+        Class<?> targetClass = obj.getClass();
+        if (dictionary.isRelation(targetClass, fieldName)) {
+            transaction.setRelation(transaction, obj, fieldName, collection, requestScope);
+        }
+
```

Comment:
I feel like I should put the setRelation hook here, because only updating toOne relation calls `setValue()` function. Correct me if I'm wrong.